pkg/detector/ospkg/redhat/redhat.go

```diff
@@ -125,3 +134,12 @@ func (s *Scanner) isSupportedVersion(now time.Time, osFamily, osVer string) bool
 	}
 	return now.Before(eolDate)
 }
+
+func (s *Scanner) isFromSupportedVendor(pkg ftypes.Package) bool {
+	for _, re := range excludedVendorsRe {
+		if re.MatchString(pkg.Release) {
+			return false
+		}
```

Comment:
We don't have to use `regexp` this time. Could you use `HasSuffix`? If we get to need a complex rule, we're going to use `regexp`.

Reply:
Sure. Changed to `HasSuffix`.